templates/ressource/_delete_form.html.twig

```diff
@@ -1,4 +1,4 @@
 <form method="post" action="{{ path('ressource_delete', {'id': ressource.id}) }}" onsubmit="return confirm('Are you sure you want to delete this item?');">
     <input type="hidden" name="_token" value="{{ csrf_token('delete' ~ ressource.id) }}">
-    <button class="btn">Supprimer</button>
+    <button class="btn">Delete</button>
```

Comment:
Why change it in english, when the rest of the site is in french ??

Reply:
Hello Thomas,

Sorry, I just made the change.
The site is in French.

Have a good day